torch/fx/experimental/proxy_tensor.py

```diff
@@ -534,6 +534,7 @@ def make_fx(f, decomposition_table=None, tracing_mode="real"):
     def wrapped(*args):
         phs = pytree.tree_map(lambda _: fx.PH, args)  # type: ignore[attr-defined]
         fx_tracer = PythonKeyTracer()
+        fx_tracer.record_stack_traces = True
```

Comment:
@Chillee, do we want to have a flag for this?

Reply:
I think we can just always record it.